components/yaml/introduction.rst

```diff
@@ -214,30 +200,27 @@ changed using the third argument as follows::
             foo: bar
             bar: baz
 
-Invalid Types and Object Serialization
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Advanced Usage: Flags
+---------------------
 
-By default the YAML component will encode any "unsupported" type (i.e.
-resources and objects) as ``null``.
+.. versionadded:: 3.1
+    Flags were added in version 3.1 and replaced the earlier boolean
+    arguments.
 
-Instead of encoding as ``null`` you can choose to throw an exception if an invalid
-type is encountered in either the dumper or parser as follows::
+Object Parsing and Dumping
+..........................
 
-    // throw an exception if a resource or object is encountered
-    Yaml::dump($data, 2, 4, true);
-
-    // throw an exception if an encoded object is found in the YAML string
-    Yaml::parse($yaml, true);
-
-However, you can activate object support using the next argument::
+You can dump objects by using the ``DUMP_OBJECT`` flag:
```

Comment:
To enable the default PHP highlighting, the last line before the code must end with 2 double colons `::`  Instead of:

```
You can dump objects by using the ``DUMP_OBJECT`` flag:
```

Please use:

```
You can dump objects by using the ``DUMP_OBJECT`` flag::
```

Same for the rest of code listings.